README.md

```diff
@@ -49,7 +52,7 @@ How we distribute the data internally within the organisation is roughly documen
 
 ### Writing functions and workflows
 
-See the separate [document](functionality.md) on needed functionality.
+All `read_xxx()` functions in this repo read data from local folders. See the separate [document](functionality.md) on needed functionality.
```

Comment:
I think this insertion can be left out, as this heading is more general and the maint point (local folders) has been made above